### DISCUSSION TITLE
Enum syntax with both names and values

### DISCUSSION THREAD

**Discussion by 159199** [microsoft/OpenAPI.NET#983] [Q&A] [answered]
Hi.
I would like to generate json with both enum name and value to preserve e.g. this:
```
public enum MyEnum
    {
        one = 1,
        three = 3
    }
```
Is it possible? And if so, how?

**Comment by baywet** [comment, marked as answer]
Hi @159199

Thank you for using the SDK and for reaching out.

This issue sounds like it originates in the ASP.NET core implementation of OpenAPI (Microsoft.AspNetCore.OpenApi). Please reach out on [the aspnet repository](https://github.com/dotnet/aspnetcore) instead.

Alternatively, if you're using Swashbuckle in your application, you may reach out [on the swashbuckle repository](https://github.com/domaindrivendev/Swashbuckle.AspNetCore).
